### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by ogiogiovictor**
# Changes

1. Created UI for the Calculator
2. Added components 
3. Used props and PropTypes

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/calculator/calculator.component';
 import './App.css';
 
 function App() {
```
Comment (descholar-ceo) [inline src/App.js]: - [ ] The App component should be a class based component

File: src/components/calculator/calculator.component.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Screen from '../screen/screen.component';
+import Button from '../buttons/buttons.component';
+
+import './calculator.style.css';
+
+const Calculator = () => {
```
Comment (descholar-ceo) [inline src/components/calculator/calculator.component.jsx]: - [ ] The Calculator component should be a class based component

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @ogiogiovictor , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 All the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. ✔️ 
👍🏽 The project has a directory called components. ✔️ 
👍🏽 There is a `Calculator.js` file inside the components directory. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @ogiogiovictor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**